code/controllers/subsystem/shuttles/supply.dm

```diff
@@ -182,27 +182,24 @@
 
 /proc/forbidden_atoms_check(atom/A)
 	if(A)
-		if(istype(A,/mob/living))
+		if(is_type_in_list(A, blacklist))
 			return 1
-		if(istype(A,/obj))
-			if(istype(A,/obj/effect/blob))
-				return 1
-			if(istype(A,/obj/effect/spider/spiderling))
-				return 1
-			if(istype(A,/obj/item/weapon/disk/nuclear))
-				return 1
-			if(istype(A,/obj/machinery/nuclearbomb))
-				return 1
-			if(istype(A,/obj/item/device/radio/beacon))
-				return 1
-
 		for(var/thing in A)
 			if(.(thing))
 				return 1
 
 	return 0
 
-
+/var/list/blacklist = list(
```

Comment:
probably best to put this in the forbidden_atoms_check() proc, it doesn't need to be reused since blacklists are usually highly specific.